backend/app/record/record/record.py

```diff
@@ -0,0 +1,19 @@
+from __future__ import annotations
+
+from abc import ABC, abstractmethod
+
+
+class Record(ABC):
```

Comment:
Class naming is odd here. It's a store for records, not a record. `RecordStore`?